modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -61,6 +61,30 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-176607. Assert manager can edit a verified organization."
+	@priority = 5
+	test ManagerCanEditVeriifiedOrganization {
+		task ("Given the Manager user is in the Organization page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "Manager User");
+
+			EVPUtils.gotoEVP();
+		}
+
+		task ("When the manager goes to edit an organization with 'Verified' status") {
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("Then the manger is able to make changes") {
+			VPOrganization.kebabMenuOrganization(organizationName = "Organization 4");
+
+			AssertElementPresent(
```

Comment:
This only checks that the Edit option is available. Please check that the manager is able to edit the organization.
1. Navigate into the organization
2. Make edits
3. Save edits
4. View organization that the edits are applied/persist.